operator/src/lib.rs

```diff
@@ -327,7 +320,7 @@ impl SparkState {
         hash: &str,
     ) -> (Vec<Container>, Vec<Volume>) {
         // TODO: get version from controller
-        let image_name = "spark:3.0.1".to_string();
+        let image_name = self.spec.image.to_string();
```

Comment:
I'm not sure but I don't think we want to allow (at least now) arbitrary "images".
This would allow someone to use any other package (even "kafka") and that doesn't really make much sense.

@soenkeliebau

Reply:
Not sure if we want to forbid that? I thought about using custom images (e.g. a client wants to experiment and e.g. in the sparklyr case one year ago, could provide a spark_plus_some_jars:3.0.1. 
Not sure how much freedom we want to provide there. If you offer a kafka image to a spark operator, well it will not start very well anyways.

Reply:
I think I'm with Lars here, the image name is something the operator should decide. If customers want to roll out "own" stuff, they should create an operator of their own..

Reply:
Ok will fix that